code/modules/power/singularity/collector.dm

```diff
@@ -44,6 +47,25 @@
 /obj/machinery/power/rad_collector/should_have_node()
 	return anchored
 
+/obj/machinery/power/rad_collector/RefreshParts()
+	var/coefficient
+	var/efficiency
+	var/miningrate

```

Comment:
even though seemingly not required by BYOND its probably best to initialize these to 0 explicitly to show what you're doing